### ISSUE TITLE
Attributes from library

### ISSUE THREAD

**Issue by thewolfprod** [tyron12233/CodeAssist#68]
Build Version: 0.1.4 / 0.1.6 ALPHA

Can't use attributes on Custom View from the library (the compiler seems to be looking for these attributes in app, not in the library)

![Screenshot_20211216_231336](https://user-images.githubusercontent.com/55655314/146589585-59a2ebd5-6254-43fc-8591-cd29d265e5b9.jpg)
![Screenshot_20211216_231410](https://user-images.githubusercontent.com/55655314/146589602-882db46e-301c-4419-a734-9ca7cd01e04c.jpg)

**Comment by tyron12233** [comment]
you have a typo on `EpandableCircleView` it should be `ExpandableCircleView`

**Comment by thewolfprod** [comment]
Ah, I didn't notice the screenshot, and I had to change it by accident before I did the screenshot (bad keyboard). After correcting, it is still the same. I can import this library normally, but I can't use the attributes that belong to it. I also can't use code instead of attributes because there is no code that responds to the same as these attributes.

![Screenshot_20211217_192508](https://user-images.githubusercontent.com/55655314/146591230-1d718947-3381-4d67-8e10-16665d4050fe.jpg)
![Screenshot_20211217_192657](https://user-images.githubusercontent.com/55655314/146591234-8e02b9f7-a1f0-49c8-b062-752b9922d9a1.jpg)
